go.mod

```diff
@@ -43,16 +43,16 @@ require (
 	k8s.io/cluster-registry v0.0.6
 	k8s.io/klog/v2 v2.90.1
 	k8s.io/kube-aggregator v0.25.9
-	k8s.io/kube-openapi v0.0.0-20230109183929-3758b55a6596
+	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
 	k8s.io/kubectl v0.25.9
-	k8s.io/kubernetes v1.25.9
-	k8s.io/utils v0.0.0-20221108210102-8e77b1f39fe2
+	k8s.io/kubernetes v1.27.3
```

Comment:
i don't know if we want to make this jump to k8s 1.27 yet. I think we would also want to bump the other k8s.io versions as well.